apps/frontend/src/teams/TeamList.tsx

```diff
@@ -22,18 +25,16 @@ const Page: React.FC = () => {
 
   if (teams) {
     return (
-      <>
+      <NetworkListPage>
         {teams.map((team) => {
```

Comment:
Aware this is a draft, but just to make sure it comes early enough:
This needs to be in a template (similar to the profiles structure). This is to keep the presentation out of the frontend and make sure everything visual can be viewed in Storybook in the same way as in the frontend.